contracts/mainnet/connectors/fluid-dex/main.sol

```diff
@@ -0,0 +1,95 @@
+//SPDX-License-Identifier: MIT
+pragma solidity ^0.8.2;
+
+/**
+ * @title Fluid Dex.
+ * @dev Dex.
+ */
+
+import {Events} from "./events.sol";
+import {IFluidDex} from "./interface.sol";
+import {TokenInterface} from "../../common/interfaces.sol";
+import {Basic} from "../../common/basic.sol";
+
+abstract contract FluidDex is Basic, Events {
+    /**
+     * @dev Returns Eth address
+     */
+    function getEthAddr() internal pure returns (address) {
+        return 0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE;
+    }
+
+    /**
+     * @dev Deposits in Fluid Dex
+     * @param dex_ Fluid dex address
+     * @param token0Amt_ The amount of token0 to deposit
+     * @param token1Amt_ The amount of token1 to deposit
+     * @param minSharesAmt_ The minimum amount of shares the user expects to receive
+     * @param estimate_ If true, function will revert with estimated shares without executing the deposit
+     */
+    function depositInDex(
+        address dex_,
+        uint256 token0Amt_,
+        uint256 token1Amt_,
+        uint256 minSharesAmt_,
+        bool estimate_
```

Comment:
@shriyatyagii We can remove this and pass always true

Reply:
Removed